cmd/bosun/sched/notify.go

```diff
@@ -209,3 +211,51 @@ func (s *Schedule) AddNotification(ak expr.AlertKey, n *conf.Notification, start
 	}
 	s.Notifications[ak][n.Name] = started
 }
+
+type actionNotificationData struct {
+	States     []*State
+	User       string
+	Message    string
+	ActionType string
```

Comment:
I was having trouble stringing it in the template. That should be possible though.